tsc/data/translations/fr.po

```diff
@@ -1,1605 +1,1728 @@
-# This file is distributed under the same license as the Secret Maryo Chronicles package.
-#
-# Copyright © 2008 Tester <joelekiffeur@yahoo.fr>
-# Copyright © 2008 Dylan Aïssi <bob.dybian@gmail.com>
-#
-# Translation of Secret Maryo Chronicles to French
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR The TSC Contributors
+# This file is distributed under the same license as the PACKAGE package.
+# 
+# Translators:
+# Benjamin Leduc <giby_the_kid@yahoo.fr>, 2015
 msgid ""
 msgstr ""
-"Project-Id-Version: Secret Maryo Chronicles\n"
+"Project-Id-Version: The Secret Chronicles of Dr. M., TSC\n"
```

Comment:
This one must be "Secretchronicles" or there’s a risk of Gettext not finding this file, because "Secretchronicles" is the Gettext module name.